libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -286,7 +286,7 @@ SymbolicArrayVariable::SymbolicArrayVariable(
 		std::make_shared<TupleSort>(
 			"array_length_pair",
 			std::vector<std::string>{"array", "length"},
-			std::vector<SortPointer>{m_sort, SortProvider::intSort}
+			std::vector<SortPointer>{m_sort, SortProvider::sintSort}
```

Comment:
Why signed?

Reply:
Because bug

Reply:
Oh wow, didn't expect that!

Reply:
I don't really remember why I wrote signed, so I guess bug

Reply:
Good catch anyway :)